### PR TITLE
Proxy uspv and powless

### PR DESCRIPTION
**Pull request by metalicjames**
Similarly to #202, proxy Wallits with the `--proxywallet` command line option.

### CODE REVIEW

File: uspv/init.go [hunk 1]
```diff
@@ -31,7 +31,6 @@ func (s *SPVCon) parseRemoteNode(remoteNode string) (string, string, error) {
 		}
 		// only ipv4 clears this since ipv6 has colons
 		conMode = "tcp4"
-		log.Println("HERE")
```
Comment (Varunram) [inline uspv/init.go]: thanks for removing this lol

**Comment by adiabat** [comment]
Cool - I have a suggestion: how about:

`--proxy` socks5 proxy for all network communication

`--litproxy` socks5 proxy for communication to other lit nodes

`--chainproxy` socks5 proxy for communication to blockchain nodes (uspv, powless, etc)

then the default would do both, and people who need different proxys or no proxys for the separate functions can use the specific options.

**Comment by metalicjames** [comment]
retest this please

**Comment by metalicjames** [comment]
retest this please

**Comment by metalicjames** [comment]
retest this please

**Pull request merged**